cmds/core/yes/yes.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"strings"
+)
+
+func main() {
+	flag.Parse()
+	args := flag.Args()
+	yes := "y"
+	if len(args) > 0 {
+		yes = strings.Join(args, " ")
+	}
+	for {
+		if _, err := fmt.Println(yes); err != nil {
+			log.Fatalf("Println got %v error, want nil", err)
```

Comment:
thinking about this, I was wrong about adding a log.Fatal. You can try this experiment
yes | echo hi
echo does not read from stdin so yes write to stdout fails first time. There is no error printed from yes.
yes should silently exit.

you can see the error with this command
strace -e trace=write yes | echo hi
write(1, "y\ny\ny\ny\ny\ny\ny\ny\ny\ny\ny\ny\ny\ny\ny\ny\n"..., 8192) = -1 EPIPE (Broken pipe)

yes does not print an error, just exits, so I gave you bad guidance, just change that log.Fatalf line to break.
sorry!

Reply:
I see what you are saying. I fixed it.